### DISCUSSION TITLE
DELPH-IN Summit 2023: Save the date!

### DISCUSSION THREAD

**Discussion by emilymbender** [delph-in/docs#90] [Team Posts]
Dear all,

We are pleased to announce that the 2023 Summit will be hosted by UDC in A Coruña, Spain, from June 26-30. You can find information about the event at this wiki page:

https://github.com/delph-in/docs/wiki/GaliciaTop

We are looking forward to seeing many of you there in person + many virtually as well. And a huge thank you to Olga for hosting us!

We will be sending further messages asking for program input/informal registration closer to the event. In the meantime, to reassure us that this GitHub-based communications channel is functioning well, we'd appreciate a signal that this message has been received, e.g. a react emoji on the thread on GitHub...

Finally, please consider who else at your site might be interested in participating and let them know too. 

Emily, 
for the Standing Committee

**Comment by danflick** [comment]
Thumbs-up emoji.

**Comment by anncopestake** [comment]
:react

**Comment by lascarid** [comment]
Got the email loud and clear!  I hope to attend virtually.  Alex.

**Comment by osenova** [comment]
Got it well. Hope this time to be able to participate even if virtually.

**Comment by lmorgadodacosta** [comment]
🐼
